CONTRIBUTING.md

```diff
@@ -9,3 +9,25 @@
 3. Introduce your change. If it's a new feature then write a test for it as well.
 4. Make sure that tests are passing.
 5. Push to your fork and submit a pull request.
+
+#### Docker for development
+
+Alternatively you can use Docker for the development setup. This requires Docker
+and Docker Compose installed.
+
+```
+make build
+make bundle
```

Comment:
This gives me:

```
➜  tiddle git:(c9566a0) make bundle
docker-compose run --rm library bundle install
Creating network "tiddle_default" with the default driver
Creating volume "tiddle_vendor" with default driver
Creating volume "tiddle_redis" with default driver
Creating tiddle_redis_1 ... done
Creating tiddle_library_run ... done
Could not locate Gemfile
ERROR: 10
make: *** [bundle] Error 10
```

We need to add `BUNDLE_GEMFILE=gemfiles/rails7.0.gemfile` or something similar in `docker-compose.yml`

Reply:
Yeah, I didn't realize this because I was using a local Gemfile 😅 

I removed my local Gemfile and applied all your changes above. It'll use ruby 3.1.2 by default along with the Gemfile for rails 7.

In my computer, `make bundle` and `make test` both ran successfully!